src/main/java/ToDos.java

```diff
@@ -0,0 +1,9 @@
+public class ToDos extends Task{
```

Comment:
Task should be followed by a whitespace
```suggestion
public class ToDos extends Task {
```